FirebaseCore.podspec

```diff
@@ -53,7 +53,7 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   # Remember to also update version in `cmake/external/GoogleUtilities.cmake`
   s.dependency 'GoogleUtilities/Environment', '~> 8.0'
   s.dependency 'GoogleUtilities/Logger', '~> 8.0'
-  s.dependency 'FirebaseCoreInternal', '~> 11.0'
+  s.dependency 'FirebaseCoreInternal', '~> 11.4'
```

Comment:
`FirebaseCoreInternal` added API in 11.4 that FirebaseCore depends on so FirebaseCore's dependency on FirebaseCoreInternal needs to be updated like this.